### DISCUSSION TITLE
Cannot use filter on custom site columns

### DISCUSSION THREAD

**Discussion by subbareddyvs** [microsoft-search/pnp-modern-search#1492] [Q&A]
Hi, I am follwing exactly same as in this article - https://docs.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search

However I do not get to filter from the search results, no filter options shown even though the results shown have differnt custom column properties.  Filter on Filetype is good, but MyCustomColumnForSearch does not shown up in Filter webpart.
I have tried mapping custom column both at site level and tenant level, but nothing works.

**Comment by wobba** [comment]
Site columns -> managed properties are not refinable, so you cannot use them as filter managed properties. If you want to use values as a filter you need to map crawled properties `ows_columnname` to e.g. `RefinableString100`.

**Reply by subbareddyvs** [reply]
Hi wobba,
yes i have mapped crawled property ows_columnname to RefinableString01, however cannot get to filter on this column.  Can you please help how to fix this.

**Reply by subbareddyvs** [reply]
![image](https://user-images.githubusercontent.com/63510269/141315838-80f3380b-bb21-4a65-b894-d4f596ea2140.png)

**Reply by wobba** [reply]
Ensure data is indexed after you did the mapping. And that mapping happened at the right levels (tenant/site) depending on usage.
